src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5369,6 +5369,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NullabilityMismatchInTypeParameterConstraint_Title" xml:space="preserve">
     <value>The type cannot be used as type parameter in the generic type or method. Nullability of type argument doesn't match constraint type.</value>
   </data>
+  <data name="WRN_MissingNonNullTypesContextForAnnotation" xml:space="preserve">
+    <value>The annotation for nullable reference types can only be used in code within a '[NonNullTypes(true)]' context.</value>
```

Comment:
>can only be used [](start = 55, length = 16)

This diagnostic is a warning only - should the text say "should only be used" rather than "can"?